plugins/@grouparoo/snowflake/__tests__/fixtures/integration/cli-generate-from.js

```diff
@@ -0,0 +1 @@
+const nock = require("nock");
```

Comment:
Here's another one where nothing happens

Reply:
I decided to leave them here because it prevents any calls. if we didn't do it, it might be talking to the remove server and we didn't know.